src/rust/pal/functions.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+use crate::pal::data_structures::SockAddrIn;
+
+//======================================================================================================================
+// Windows functions
+//======================================================================================================================
```

Comment:
And again here, group functions by type, rather than platform.

Reply:
@BrianZill I agree, but I asked @anandbonde to get first this merged in, and then he will re-work this source tree.

Reply:
Agreed, Pedro and I had planned to do this in subsequent PR(s). [As per the next steps listed in the PR description above: (1) Refactor the PAL to make platform/subsystem specific hierarchy in the directory structure.]